public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
### Your Pull Request Summary
![not descriptive](https://user-images.githubusercontent.com/93039359/204906987-334f9468-dfcd-4427-b218-4501d4894cd8.PNG)

### Descriptive Pull Request Summary
![Descriptive Summary Initialize Math Magician Application](https://user-images.githubusercontent.com/93039359/204907005-e6136080-7211-4505-921e-defb01be0ffb.PNG)

- [ ] You can see that your summary is not descriptive enough. Kindly update your summary and make it descriptive to make your pull request professional 😊

**Hint: Kindly update it by click on the three dots on the upper left corner of this summary box  👍**